src/sentry/demo/data_population.py

```diff
@@ -1228,6 +1385,13 @@ def populate_generic_error(
             update_context(local_event, platform=project.platform)
             self.fix_error_event(local_event)
             self.safe_send_event(local_event)
+
+            if project.slug in alert_platforms:
+                if project.slug not in self.timestamps_by_project:
+                    self.timestamps_by_project[project.slug] = [timestamp]
```

Comment:
@roggenkemper why not push this functionality into `safe_send_event` and check the type being `error` there?